slides/advanced-05-iterative.qmd

```diff
@@ -38,6 +38,13 @@ options(width = 200)
 ggplot2::theme_set(ggplot2::theme_bw())
 
 load("bayes_opt_calcs.RData")
+
+options(
+  cli.width = 70,
+  ggplot2.discrete.fill = c("#1b9e77", "#d95f02"),
+  ggplot2.discrete.colour = c("#1b9e77", "#d95f02")
+)
```

Comment:
I rendered everything (via the index.qmd) to see if this would affect the default colors for the intro course. It seems like that is okay. But I also didn't see these pop up in this advanced deck at first. They are only slightly different from the green/tan for the intro deck, right?

Reply:
Yes, they should only affect these slides; they are RColorBrewer::Set2